build.gradle

```diff
@@ -41,14 +42,17 @@ spotless {
   }
 }
 
+// Use the osdetector-gradle-plugin
+apply plugin: "com.google.osdetector"
```

Comment:
are we actually using this? if so where/how ?

Reply:
yes, it is used [here](https://github.com/linkedin/kafka/blob/0bd73e7e126a7448bad76478a0180fff0e777d6c/gradle/dependencies.gradle#L144)